pypeit/reduce.py

```diff
@@ -874,6 +874,12 @@ def find_objects_pypeline(self, image, std_trace=None,
         # Instantiate the specobjs container
         sobjs = specobjs.SpecObjs()
 
+        # Masking options
+        if self.par['reduce']['skysub']['mask_by_boxcar']:
+            boxcar_rad_pix = self.par['reduce']['extraction']['boxcar_radius'] / self.get_platescale(None),
+        else:
+            boxcar_rad_pix = None
+
         # Loop on slits
```

Comment:
boxcar_rad_pix = self.par['reduce']['extraction']['boxcar_radius'] / self.get_platescale(None) if self.par['reduce']['skysub']['mask_by_boxcar'] else None

Reply:
Please comment.

Reply:
I prefer it the other way